components/filter/README.md

```diff
@@ -1,51 +1,194 @@
+  import '@brightspace-ui/core/components/filter/filter-dimension-set.js';
+  import '@brightspace-ui/core/components/filter/filter-dimension-set-value.js';
+</script>
+<!-- docs: start hidden content -->
+<script>
+  window.addEventListener('load', function () {
+    var demoElem = document.querySelector('#demo-element');
+    if (!demoElem.hasAttribute('data-first-load')) return;
+    setTimeout(() => {
+		var filter = document.querySelector('d2l-filter');
+		filter.opened = true;
+    }, 100);
+  });
+</script>
+<style>
+	.flex-div {
+		display: flex;
+		flex-direction: column;
+		justify-content: center;
+	}
+</style>
+<!-- docs: end hidden content -->
+<div class="flex-div">
+	Single Dimension
+	<d2l-filter>
+		<d2l-filter-dimension-set key="course" text="Course">
+			<d2l-filter-dimension-set-value key="art" text="Art"></d2l-filter-dimension-set-value>
+			<d2l-filter-dimension-set-value key="astronomy" text="Astronomy" selected></d2l-filter-dimension-set-value>
+			<d2l-filter-dimension-set-value key="biology" text="Biology"></d2l-filter-dimension-set-value>
+			<d2l-filter-dimension-set-value key="chemistry" text="Chemistry"></d2l-filter-dimension-set-value>
+		</d2l-filter-dimension-set>
+	</d2l-filter>
+</div>
+<div class="flex-div">
+	Multi-Dimensional
+	<d2l-filter>
+		<d2l-filter-dimension-set key="course" text="Course">
+			<d2l-filter-dimension-set-value key="art" text="Art"></d2l-filter-dimension-set-value>
+			<d2l-filter-dimension-set-value key="astronomy" text="Astronomy" selected></d2l-filter-dimension-set-value>
+			<d2l-filter-dimension-set-value key="biology" text="Biology"></d2l-filter-dimension-set-value>
+			<d2l-filter-dimension-set-value key="chemistry" text="Chemistry"></d2l-filter-dimension-set-value>
+		</d2l-filter-dimension-set>
+		<d2l-filter-dimension-set key="semester" text="Semester" selection-single>
+			<d2l-filter-dimension-set-value key="fall" text="Fall"></d2l-filter-dimension-set-value>
+			<d2l-filter-dimension-set-value key="winter" text="Winter" selected></d2l-filter-dimension-set-value>
+			<d2l-filter-dimension-set-value key="spring" text="Spring"></d2l-filter-dimension-set-value>
+			<d2l-filter-dimension-set-value key="summer" text="Summer"></d2l-filter-dimension-set-value>
+		</d2l-filter-dimension-set>
+	</d2l-filter>
+</div>
+``` 
+
+### Accessibility
+The filter will announce changes to filter selections, search results, and when filters are being cleared. It is up to the consumer to then announce when these changes have propogated and resulted in new/loaded/updated data on the page. This is very important for screenreader users who are not able to visually see the page changing behind the filter control as selections are made.
+
+<!-- docs: start hidden content -->
+### Properties
 
 | Property | Type | Description |
-|--|--|--|
+|---|---|---|
 | `disabled` | Boolean, default: `false` | Disables the dropdown opener for the filter |
```

Comment:
Whoops, guess I should have added `opened` here.  Are these tables temporary until the Daylight site is live?

Reply:
Hmm we decided to leave them in the readme for now.. Not 100% sure on the future of property tables. I can add the `opened` line here